service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GpcStructuredResponseObject.java

```diff
@@ -0,0 +1,13 @@
+package uk.nhs.adaptors.gp2gp.gpc;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class GpcStructuredResponseObject {
```

Comment:
All of our storage objects will be wrapped with this exact same metadata. Maybe this should be a `StorageWrapperObject` in the `common.storage` package so it can be re-used?